ibis/backends/sql/compilers/clickhouse.py

```diff
@@ -97,7 +95,6 @@ class ClickHouseCompiler(SQLGlotCompiler):
         ops.Last: "anyLast",
         ops.Ln: "log",
         ops.Log10: "log10",
-        ops.MapContains: "mapContains",
```

Comment:
is this being handled elsewhere?

Reply:
Yep, there's a new `visit_MapContains` implementation near the end of the file.

Reply:
:face_with_peeking_eye: